gpiozero/boards.py

```diff
@@ -178,9 +200,96 @@ def _blink_device(self, on_time, off_time, fade_in_time, fade_out_time, n, fps=5
                 break
 
 
+class LEDBarGraph(LEDCollection):
+    """
+    Extends :class:`CompositeDevice` to control a line of LEDs representing a
+    bar graph. Positive values (0 to 1) light the LEDs from first to last.
+    Negative values (-1 to 0) light the LEDs from last to first.
```

Comment:
Should this be changed to "Negative values (0 to -1) ..." to reflect that increasingly-larger negative values light up more LEDS "from last to first" ?